src/app/services/game.service.ts

```diff
@@ -0,0 +1,94 @@
+import { Tile } from '../models/tile';
+import { Player } from '../models/player';
+
+export class GameService {
+  tiles: Tile[];
+
+  players: Player[];
+  occupiedBlock: number;
+
+  constructor() {
+    this.initTiles();
+    this.initPlayers();
+  }
+
+  initTiles(): void {
+    this.tiles = [];
+    this.occupiedBlock = 0;
+    for (let i = 0; i < 9; i++) {
+      const tile = new Tile(true, '');
+
+      this.tiles.push(tile);
+    }
+  }
+
+  initPlayers(): void {
+    this.players = [];
+    this.players.push(new Player(true, 'X'));
+    this.players.push(new Player(false, 'O'));
+  }
+
+  makeSign(item: Tile): void {
+    const playerSign = this.checkPlayerTurn();
+    console.log('turn -> ' + playerSign);
+    if (item.empty) {
+      item.value = playerSign;
+      this.occupiedBlock++;
+      item.empty = false;
+    }
+    this.checkIfWin(this.tiles);
+  }
+
+  checkPlayerTurn(): string {
+    if (this.players[0].turn) {
+      this.players[0].turn = false;
+      this.players[1].turn = true;
+      return this.players[0].sign;
+    } else if (this.players[1].turn) {
+      this.players[1].turn = false;
+      this.players[0].turn = true;
+      return this.players[1].sign;
+    }
+  }
+
+  checkIfWin(tiles: Tile[]): void {
```

Comment:
It is very long method. It could be replaced with some algorithm that will verify each line and diagonals. It will be cleaner this way.